cri/v1alpha2/cri.go

```diff
@@ -180,6 +183,20 @@ func NewCriManager(config *config.Config, ctrMgr mgr.ContainerMgr, imgMgr mgr.Im
 		return nil, fmt.Errorf("failed to create sandbox meta store: %v", err)
 	}
 
+	c.ContainerStore, err = meta.NewStore(meta.Config{
```

Comment:
could you provide more detail about this design? I think it should be described in the PR.